s2core/src/main/scala/com/kakao/s2graph/core/storage/IndexEdgeSerializable.scala

```diff
@@ -0,0 +1,58 @@
+package com.kakao.s2graph.core.storage
```

Comment:
move package from com.kakao.s2graph.core.storage.hbase to  com.kakao.s2graph.core.storage